tests/Nox.Yaml.Tests/TestDesigns/Nox/Schemas/domainCommand.json

```diff
@@ -1903,20 +1903,7 @@
                     "supportedFileFormats": {    
                       "oneOf": [
                         {
-                          "type": "array",
-                          "enum": [
-                            "all",
-                            "csv",
-                            "doc",
-                            "docx",
-                            "eml",
-                            "pdf",
-                            "ppt",
-                            "pptx",
-                            "txt",
-                            "xls",
-                            "xlsx"
-                          ]
+                          "type": "array"
```

Comment:
Why is this removed again, do you know @dejobratic ?

Reply:
I think it's related to this PR: https://github.com/NoxOrg/Nox/pull/1352.